### ISSUE TITLE
Add a way to fake bluetooth being enabled in waydroid.

### ISSUE THREAD

**Issue by jamie-pate** [waydroid/waydroid#1378]
### Is your feature request related to a problem? Please describe.

Trying to run this apk because the unifi wifi router's web app is terrible: https://apkpure.com/amplifi-wifi/com.ubnt.unifihome

But the onboarding screen won't continue until I enable bluetooth, which doesn't seem to be possible yet.
Whenever I use the settings to enable bluetooth it just turns off immediately.

### Describe the solution you'd like.

Either just fake bluetooth being enabled when you turn it on, or add a waydroid prop `persist.waydroid.fake_bluetooth` which accepts a comma separated list of package names for which the system will always appear as if bluetooth is enabled.
This approach is already used for wifi with `persist.waydroid.fake_wifi`

### Describe alternatives you've considered.

forwarding the whole bluetooth stack or adding some kind of bluetooth passthrough, but these are much more complicated.

### Additional context

I suppose some stubs would need to be written to fake handle bluetooth api calls and just always return an empty list of devices, etc

**Comment by mamolas** [comment]
real Bluetooth support would make Waydroid a ton better, I can pair and use bluetooth devices flawlessly on Androidx86, unfortunately they stopped updates and development 3+ years ago then I found Waydroid but it lacks this key feature

**Comment by RocketKevQt** [comment]
yes, there is a ton of device that provide support for windows and android through bluetooth but no linux support, adding bluetooth on waydroid would fix some of the issues us as linux users face using those devices
